### DISCUSSION TITLE
How many structures are in the 2022 NSI dataset?

### DISCUSSION THREAD

**Discussion by thwllms** [HydrologicEngineeringCenter/NSI#17] [Q&A] [answered]
Was looking around at different nationwide buildings datasets. Curious about NSI's count.
* USA structures: 131,811,972
https://fema.maps.arcgis.com/home/item.html?id=0ec8512ad21e4bb987d7e848d14e7e24#data
https://gis-fema.hub.arcgis.com/pages/usa-structures
* Bing: 129,591,852
https://github.com/Microsoft/USBuildingFootprints

**Comment by HenryGeorgist** [comment]
NSI is built based on a variety of inputs including both Bing Footprints and USA Structures footprints. There is no perfect dataset out there, but rather various datasets that fill needs at different qualities. In the NSI you can look at the field ftprntsrc that identifies which footprint dataset was used to place the point. 
https://www.hec.usace.army.mil/confluence/nsi/technicalreferences/latest/technical-documentation#id-.TechnicalDocumentationv2022-NSIPublicFields

I will get a count of the national dataset and post it.

**Comment by trietmnj** [comment, marked as answer]
@thwllms 123,017,988 is the count for this latest 2022 release.
